packages/sql-commands/0.1.0/packages.yml

```diff
@@ -0,0 +1,30 @@
+matches:
+  - trigger: ":CDB:"
+    replace: "CREATE DATABASE"
+
+  - trigger: ":ADB:"
+    replace: "ALTER DATABASE"
+
+  - trigger: ":II:"
+    replace: "INSERT INTO"
+
+  - trigger: ":SF:"
+    replace: "SELECT * FROM"
```

Comment:
This one could utilise a [Cursor Hints](https://espanso.org/docs/matches/basics/#cursor-hints) `$|$` facility.